portal_recaptcha/views/portal_mixin.xml

```diff
@@ -4,17 +4,9 @@
     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 -->
 <odoo>
-    <template
-        id="assets_editor_recaptcha"
-        name="Google reCAPTCHA"
-        inherit_id="portal.frontend_layout"
-    >
-        <xpath expr="//main" position="inside">
-            <script src="https://www.google.com/recaptcha/api.js" />
-        </xpath>
-    </template>
 
     <template id="recaptcha_widget" name="reCAPTCHA widget">
+        <script src="https://www.google.com/recaptcha/api.js" />
```

Comment:
is it allowed to have a `script` element there?

Reply:
I don't know if it's allowed but it works ;)